runtime/str.go

```diff
@@ -918,6 +933,57 @@ func strInterpolate(f *Frame, format string, values *Tuple) (*Object, *BaseExcep
 	return NewStr(buf.String()).ToObject(), nil
 }
 
+func strIsAlNum(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "isalnum", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	if len(s) == 0 {
+		return GetBool(false).ToObject(), nil
+	}
+	for i := range s {
+		if !isAlNum(s[i]) {
+			return GetBool(false).ToObject(), nil
+		}
+	}
+
+	return GetBool(true).ToObject(), nil
+}
+
+func strIsAlpha(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "isalpha", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	if len(s) == 0 {
+		return GetBool(false).ToObject(), nil
+	}
+	for i := range s {
+		if !isAlpha(s[i]) {
+			return GetBool(false).ToObject(), nil
+		}
+	}
+
+	return GetBool(true).ToObject(), nil
+}
+
+func strIsDigit(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "isdigit", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	if len(s) == 0 {
+		return GetBool(false).ToObject(), nil
```

Comment:
I think it's more suitable for just using the global bool instance `True` / `False.`